From 4c55db04cb45046953dd7075b4588c4860c4c500 Mon Sep 17 00:00:00 2001
From: "Ye.Li" <B37916@freescale.com>
Date: Thu, 17 Jul 2014 11:50:30 +0800
Subject: [PATCH 0858/1543] ENGR00323026 iMX6SX:MCC Disable ADC1 and ADC2 in
 MCC DTS

commit e68dca2c98b61317fae0c93823cf9e63a84480bd from
git://git.freescale.com/imx/linux-2.6-imx.git

M4 has occupied the ADC1 and ADC2 in RDC, must disable them in DTS
to avoid kernel boot panic.

Signed-off-by: Ye.Li <B37916@freescale.com>
(cherry picked from commit 0588a0133d649ebdab32d7072c64a051845191f7)
---
 arch/arm/boot/dts/imx6sx-sdb-m4.dts |    8 ++++++++
 1 files changed, 8 insertions(+), 0 deletions(-)

diff --git a/arch/arm/boot/dts/imx6sx-sdb-m4.dts b/arch/arm/boot/dts/imx6sx-sdb-m4.dts
index 2d70685..aa52ca7 100644
--- a/arch/arm/boot/dts/imx6sx-sdb-m4.dts
+++ b/arch/arm/boot/dts/imx6sx-sdb-m4.dts
@@ -27,3 +27,11 @@
 &uart2 {
 	status = "disabled";
 };
+
+&adc1 {
+        status = "disabled";
+};
+
+&adc2 {
+        status = "disabled";
+};
-- 
1.7.5.4

